### PR TITLE
9.x upgrade tests are added

### PR DESCRIPTION
**Pull request by rkhozinov**
Run update and update that

### CODE REVIEW

File: servers/partner-ci/9.x/upgrade.yaml [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+    testgroup_parameter_type: string
+    testgroup_value_type: default
+    testgroup_value: '{testgroup}'
+
+    testgroup:
+      - install_mu_no_ha_base_negative
+      - install_mu_no_ha_failover
+      - install_mu_ironic_ceilometer
+      - install_mu_ha
```
Comment (naumvd95) [inline servers/partner-ci/9.x/upgrade.yaml:97]: where is that testgroup used?

Comment (naumvd95) [reply servers/partner-ci/9.x/upgrade.yaml:97]: +      - install_mu_ha

File: servers/partner-ci/9.x/upgrade.yaml [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+      - common-scm:
+          scm_submodule_disabled: '{scm_submodule_disabled}'
+          scm_wipe_workspace: '{scm_wipe_workspace}'
+          gerrit_repo: '{gerrit_repo}'
+          skip_vote_successfull: '{skip_vote_successfull}'
+          skip_vote_failed: '{skip_vote_failed}'
+    wrappers:
+      - test-wrapper
+    properties:
+     - test-property
```
Comment (naumvd95) [inline servers/partner-ci/9.x/upgrade.yaml]: 3-x indent

**Review by naumvd95 — COMMENTED** [review]
1) read comments

File: servers/partner-ci/9.x/upgrade-nightly.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+- runner-condition
+- multijob:
+    name: 'Smoke test for fuel {fuel_release}'
+    condition: SUCCESSFUL
+    projects:
+     - name: '{fuel_release}.{plugin-name}.install_mu_ha'
```
Comment (rkhozinov) [inline servers/partner-ci/9.x/upgrade-nightly.yaml:6]: here

File: servers/partner-ci/9.x/upgrade.yaml (hunk 1 above)
Comment (rkhozinov) [reply servers/partner-ci/9.x/upgrade.yaml:97]: line 6

**Comment by rkhozinov** [comment]
recheck

File: servers/partner-ci/9.x/upgrade-nightly.yaml (hunk 3 above)
Comment (naumvd95) [reply servers/partner-ci/9.x/upgrade-nightly.yaml:6]: ok

**Review by naumvd95 — APPROVED** [review]

**Pull request merged**